docs/demos/TrafficManagerProfile/session-build-multi-clsuter-app/README.md

```diff
@@ -0,0 +1,221 @@
+                matchLabels:
+                  fleet.azure.com/location: eastus2euap
+        jsonPatchOverrides:
+          - op: add
+            path: /metadata/annotations
+            value:
+              {"service.beta.kubernetes.io/azure-dns-label-name":"fleet-${MEMBER-CLUSTER-NAME}-eastus2euap"}
+```
+> Note: Please update the dns label name to match your specific requirements, and the "fleet-${MEMBER-CLUSTER-NAME}-eastus2euap" may be not available.
+
+
+Apply the configuration:
+```sh
+kubectl apply -f eastus2euap/ro-nginx-service-eastus2euap.yaml
+```
+---
+
+## Step 4: Define clusterResourcePlacement
+Use **clusterResourcePlacement** to deploy Nginx across multiple clusters:
+
+> Note: test file located [here](../testfiles/crp.yaml).
+
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1
+kind: ClusterResourcePlacement
+metadata:
+  name: crp-team-a
+spec:
+  resourceSelectors:
+    - group: ""
+      kind: Namespace
+      name: team-a-nginx
+      version: v1
+  policy:
+    placementType: PickAll
+    affinity:
+      clusterAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+```
+
+Apply the configuration:
+```sh
+kubectl apply -f crp.yaml
+```
+
+---
+
+## Step 5: Configure Azure Traffic Manager
+Define **Traffic Manager Profile** and **Traffic Manager Backend** for global traffic distribution once the resources are available on the members.
+
+### Traffic Manager Profile
+> Note: test file located [here](../testfiles/nginx-profile.yaml).
+
+```yaml
+apiVersion: networking.fleet.azure.com/v1beta1
+kind: TrafficManagerProfile
+metadata:
+  name: nginx-profile
+  namespace: team-a-nginx
+spec:
+  resourceGroup: "your-fleet-atm-rg"
+  monitorConfig:
+    port: 80
+```
+
+> Note: Please update the profile or namespace name if the dns name of the profile is not available.
+
+### Traffic Manager Backend
+
+> Note: test file located [here](../testfiles/nginx-backend-eastus2euap.yaml).
+```yaml
+apiVersion: networking.fleet.azure.com/v1beta1
+kind: TrafficManagerBackend
+metadata:
+  name: nginx-backend-eastus2euap
+  namespace: team-a-nginx
+spec:
+  profile:
+    name: "nginx-profile"
+  backend:
+    name: "nginx-service-eastus2euap"
+  weight: 100
+```
+
+Apply the configurations:
+```sh
+kubectl apply -f nginx-profile.yaml
+kubectl apply -f testfiles/nginx-backend-eastus2euap.yaml
+```
+
+---
+
+## **Conclusion 🎯**
+By completing these steps, you now have:
+- Deployed Nginx across multiple clusters.
+- Assigned unique DNS labels per cluster.
+- Published the service across clusters using **ServiceExport**.
+- Integrated **Azure Traffic Manager** for intelligent global traffic routing.
+
+Now, users can access the **team-a-nginx-nginx-profile.trafficmanager.net** endpoint, and traffic will automatically distribute between clusters, with failover handling in case of failures.

```

Comment:
Hi Zhiying! Just a nit: we might need some instructions for helping people verify the setup? This is not a blocker though.